### ISSUE TITLE
Installation hangs with "testing if installed package can be loaded from temporary location"

### ISSUE THREAD

**Issue by hernanmd** [dyerlab/gstudio#12]
Installation hangs while testing package loading of "dyerlab/gstudio" using macOS highSierra with latest R as of 04-06-2020 (4.0.0). 
The expressions used for installation are :

```R
install.packages( c("RgoogleMaps",
                    "geosphere",
                    "proto",
                    "sampling",
                    "seqinr",
                    "spacetime",
                    "spdep"), 
                  dependencies=TRUE )
install.packages("devtools")
library(devtools)
install_github("dyerlab/popgraph")
install_github("dyerlab/gstudio")
```

**Comment by dyerlab** [comment]
So can you install these packages by themselves?  Which one stops and what
are the error messages?
--
Rodney J. Dyer, PhD
Director, Center for Environmental Studies
<https://ces.vcu.edu/>
Virginia Commonwealth University
http://dyerlab.com

**Comment by hernanmd** [comment]
There are no error messages, installation hangs meaning R does not respond anymore and I should terminate R manually from another shell command line terminal.
The line which causes the hang is:

install_github("dyerlab/gstudio")

Maybe you could output your R and package versions to check which dependencies are currently working for gstudio?

Anyway this should be reproducible with R as of 04-06-2020 (4.0.0).

**Comment by hernanmd** [comment]
To clarify, all package dependencies were installed successfully, i.e. no error messages.
The hang problem only occurs during the evaluation of :

`install_github("dyerlab/gstudio")`

**Comment by dyerlab** [comment]
So, nothing happens?  No messages, nothing?

--
Rodney J. Dyer, PhD
Director, Center for Environmental Studies
<https://ces.vcu.edu/>
Virginia Commonwealth University
http://dyerlab.com

**Comment by hernanmd** [comment]
Exactly, maybe you could let us know which package versions you're using in your working environment?

```R
ip <- as.data.frame(installed.packages()[,c(1,3:4)])
rownames(ip) <- NULL
ip <- ip[is.na(ip$Priority),1:2,drop=FALSE]
print(ip, row.names=FALSE)
```

and 

`sessionInfo()`

**Comment by barry282228** [comment]
Hi, did you solve this problem?
I got the same problem while installing mailR.

`** testing if installed package can be loaded from temporary location`

**Comment by asqatqazet** [comment]
Is this issue solved?

**Comment by dyerlab** [comment]
If I remember correctly, this may be more of a platform issue.  Do not load from temporary drives.  Am I remembering this incorrectly?

Dr. D
